src/Analyzers/CSharp/Tests/RemoveUnusedParametersAndValues/RemoveUnusedValueAssignmentTests.cs

```diff
@@ -133,6 +133,53 @@ int M()
             }.RunAsync();
         }
 
+        [WorkItem(48070, "https://github.com/dotnet/roslyn/issues/48070")]
+        [Theory, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnusedValues)]
+        [InlineData(UnusedValuePreference.DiscardVariable)]
+        [InlineData(UnusedValuePreference.UnusedLocalVariable)]
+        public async Task Initialization_ConstantValue_DontCopyLeadingTriviaDirectives(object option)
+        {
+            var source =
+@"class C {
+    void M()
+    {
+        #region
+        int value = 3;
+        #endregion
+
+        int? {|IDE0059:x|} = null;
+        int y = value + value;
+        
+        x = y;
+        System.Console.WriteLine(x);
+    }
+}";
+            var fixedSource =
+@"class C {
+    void M()
+    {
+        #region
+        int value = 3;
+
```

Comment:
spacing is def a bit weird here.would ideally have the #endregion stay where it was.  is that not doable?